Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -576,254 +708,79 @@
-RECODEBUGEventContent.outputCommands.extend(SimGeneralFEVTDEBUG.outputCommands)
-RECODEBUGEventContent.outputCommands.extend(SimTrackerDEBUG.outputCommands)
-
-from Configuration.ProcessModifiers.premix_stage2_cff import premix_stage2
-from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
-(premix_stage2 & phase2_tracker).toModify(FEVTDEBUGHLTEventContent, outputCommands = FEVTDEBUGHLTEventContent.outputCommands+[
-    'keep *_*_Phase2OTDigiSimLink_*'
-])
-from Configuration.Eras.Modifier_phase2_muon_cff import phase2_muon
-(premix_stage2 & phase2_muon).toModify(FEVTDEBUGHLTEventContent, outputCommands = FEVTDEBUGHLTEventContent.outputCommands+[
-    'keep *_*_GEMDigiSimLink_*',
-    'keep *_*_GEMStripDigiSimLink_*',
-    'keep *_*_ME0DigiSimLink_*',
-    'keep *_*_ME0StripDigiSimLink_*',
-])
-
-REPACKRAWSIMEventContent.outputCommands.extend(['drop FEDRawDataCollection_source_*_*',
-                                                'drop FEDRawDataCollection_rawDataCollector_*_*'])
-REPACKRAWEventContent.outputCommands.extend(['drop FEDRawDataCollection_source_*_*',
-                                                'drop FEDRawDataCollection_rawDataCollector_*_*'])
-
-
+REPACKRAWSIMEventContent.outputCommands.extend([
+	'drop FEDRawDataCollection_source_*_*',
+        'drop FEDRawDataCollection_rawDataCollector_*_*'])
+REPACKRAWEventContent.outputCommands.extend([
+	'drop FEDRawDataCollection_source_*_*',
+        'drop FEDRawDataCollection_rawDataCollector_*_*'])
 
 #from modules in Configuration.StandardSequence.Generator_cff fixGenInfo
 REGENEventContent = cms.PSet(
     inputCommands=cms.untracked.vstring(
-      'keep *',
-      'drop *_genParticles_*_*',
-      'drop *_genParticlesForJets_*_*',
-      'drop *_kt4GenJets_*_*',
-      'drop *_kt6GenJets_*_*',
-      'drop *_iterativeCone5GenJets_*_*',
-      'drop *_ak4GenJets_*_*',
-      'drop *_ak7GenJets_*_*',
-      'drop *_ak8GenJets_*_*',
-      'drop *_ak4GenJetsNoNu_*_*',
-      'drop *_ak8GenJetsNoNu_*_*',
-      'drop *_genCandidatesForMET_*_*',
-      'drop *_genParticlesForMETAllVisible_*_*',
-      'drop *_genMetCalo_*_*',
-      'drop *_genMetCaloAndNonPrompt_*_*',
-      'drop *_genMetTrue_*_*',
-      'drop *_genMetIC5GenJs_*_*'
-      )
+        'keep *',
+        'drop *_genParticles_*_*',
+        'drop *_genParticlesForJets_*_*',
+        'drop *_kt4GenJets_*_*',
+        'drop *_kt6GenJets_*_*',
+        'drop *_iterativeCone5GenJets_*_*',
+        'drop *_ak4GenJets_*_*',
+        'drop *_ak7GenJets_*_*',
+        'drop *_ak8GenJets_*_*',
+        'drop *_ak4GenJetsNoNu_*_*',
+        'drop *_ak8GenJetsNoNu_*_*',
+        'drop *_genCandidatesForMET_*_*',
+        'drop *_genParticlesForMETAllVisible_*_*',
+        'drop *_genMetCalo_*_*',
+        'drop *_genMetCaloAndNonPrompt_*_*',
+        'drop *_genMetTrue_*_*',
+        'drop *_genMetIC5GenJs_*_*')
 )
 
-def SwapKeepAndDrop(l):
```

Comment:
This function could be maintained, for possible future usages.
Maybe you can just add it back on top of this file, right after alll import's